packages/vuetify/src/components/VDivider/__tests__/__snapshots__/VDivider.spec.ts.snap

```diff
@@ -1,8 +1,8 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`VDivider should render component and match snapshot 1`] = `
-<hr role="separator"
-    aria-orientation="horizontal"
-    class="v-divider theme--light"
+exports[`VDivider should match a snapshot 1`] = `
+<hr class="v-divider  v-theme--light"
+    ariaorientation="horizontal"
```

Comment:
shouldn't the result still be
```suggestion
    aria-orientation="horizontal"
```
or is this intentional?

Reply:
It only does this in jest because I'm using camelCase. I can change it to dash case if it matters to anyone.

Reply:
It seems to be a vue issue, already resolved but not released yet
https://github.com/vuejs/vue-next/issues/583

Reply:
it does get kebab-cased in codepen https://codepen.io/jkarczm/pen/vYXMbWp?editors=1010, test-related issue?